scripts/lint-packages.js

```diff
@@ -75,69 +75,77 @@ let missingPeers = {};
 for (let pkg of packagePaths) {
   let json = packages[pkg];
   // check that all packages which depend on a package with a peerDependency also have that peerDependency
-  if (json.peerDependencies) {
-    for (let dep of Object.keys(json.peerDependencies)) {
-      for (let pkgUpstream of packagePaths) {
-        let pkgu = packages[pkgUpstream];
-        // if the upstream package doesn't also include the peerDependency, then fail
-        if (pkgu.dependencies && pkgu.dependencies[json.name] && pkgu.peerDependencies && !pkgu.peerDependencies[dep]) {
-          let key = `${pkgu.name} missing a peerDependency on ${dep}`
-          if (!missingPeers[key]) {
-            missingPeers[key] = {pkg: pkgu.name, pkgPath: pkgUpstream, missing: dep, deps: [json.name]};
-          } else {
-            missingPeers[key].deps.push(json.name);
-          }
-        }
-      }
-    }
-  }
+  // TODO verify that new yarn takes care of this automatically
```

Comment:
Was this verified already (and thus can we get rid of the commented code?)

Reply:
yeah, yarn install will (and does) warn about them

Reply:
for instance, you'll see warnings like
> ➤ YN0002: │ @react-spectrum/text@workspace:packages/@react-spectrum/text [63881] doesn't provide react-dom (p657bc), requested by react-aria-components.